python/pyspark/sql/tests/connect/test_connect_function.py

```diff
@@ -1821,6 +1821,106 @@ def test_json_functions(self):
             sdf.select(SF.to_json(SF.struct(SF.lit("a"), SF.lit("b")), {"mode": "FAILFAST"})),
         )
 
+    def test_xml_functions(self):
+        query = """
+            SELECT * FROM VALUES
+            ('<p><a>1</a></p>', '<p><a>1</a><a>2</a><a>3</a></p>', '<p><a attr="s"><b>5.0</b></a></p>'),
+            ('<p><a>0</a></p>', '<p><a>4</a><a>5</a><a>6</a></p>', '<p><a attr="t"></a></p>')
+            AS tab(a, b, c)
+            """
+        # +---------------+-------------------------------+---------------------------------+
+        # |              a|                              b|                                c|
+        # +---------------+-------------------------------+---------------------------------+
+        # |<p><a>1</a></p>|<p><a>1</a><a>2</a><a>3</a></p>|<p><a attr="s"><b>5.0</b></a></p>|
+        # |<p><a>1</a></p>|<p><a>4</a><a>5</a><a>6</a></p>|          <p><a attr="t"></a></p>|
+        # +---------------+-------------------------------+---------------------------------+
+
+        cdf = self.connect.sql(query)
+        sdf = self.spark.sql(query)
+
+        # test from_xml
+        for schema in [
+            "a INT",
+#            StructType([StructField("a", IntegerType())]),
```

Comment:
Let's probably remove commented codes.

Reply:
I would like to support this. pyspark sql works fine, but pyspark connect is failing to parse the StructType.
Any pointers would help.

Reply:
Then let's create JIRA and comment it as TODO for now. I think we need separate investigation.

e.g.
```python
        # TODO(SPARK-XXXXX): description (Please fill the actual description and JIRA number)
        for schema in [
            "a INT",
            # StructType([StructField("a", IntegerType())]),
            # StructType([StructField("a", ArrayType(IntegerType()))]),
        ]:
```